ci/init-plugin-test-service.sh

```diff
@@ -41,3 +41,4 @@ docker exec -i rmqnamesrv /home/rocketmq/rocketmq-4.6.0/bin/mqadmin updateTopic
 
 # prepare vault kv engine
 docker exec -i vault sh -c "VAULT_TOKEN='root' VAULT_ADDR='http://0.0.0.0:8200' vault secrets enable -path=kv -version=1 kv"
+
```

Comment:
We don't need to touch this file?

Reply:
No,we don't . I just run the function image in the dockerfile . And I ’ll discard the change in this file .